src/runtime_src/driver/xclng/tools/xbutil/xbutil.cpp

```diff
@@ -1479,3 +1493,168 @@ int xcldev::xclP2p(int argc, char *argv[])
 
     return ret;
 }
+
+/*
+ * m2mtest
+ */
+static int m2mtest_bank(xclDeviceHandle handle, uuid_t uuid, int bank_a, int bank_b)
+{
+    const size_t boSize = 3L * 1024 * 1024 * 1024;
+	if (xclOpenContext(handle, uuid, -1, true)) {
+		std::cout << "ERROR: Unable to lockdown xclbin" << std::endl;
+		return -EINVAL;
+	}
+
+    //Allocate boSrc
+    unsigned boSrc = xclAllocBO(handle, boSize, XCL_BO_DEVICE_RAM, bank_a);
```

Comment:
can we move these dup code (alloc bo, map and memset bo) into a helper function?